replica/schema_describe_helper.hh

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include "data_dictionary/data_dictionary.hh"
+#include "index/secondary_index_manager.hh"
+#include "schema/schema.hh"
+
+namespace replica {
+
+class schema_describe_helper : public ::schema_describe_helper {
+    data_dictionary::database _db;
+public:
+    explicit schema_describe_helper(data_dictionary::database db) : _db(db) { }
```

Comment:
Why isn't a reference to the db sufficient?
It should be kept alive by the caller, isn't it?

Reply:
`data_dictionary::database` is already a reference, a pointer in disguise. It is passed everywhere by value.